Pod/Classes/RealexComponent/HPPManager.swift

```diff
@@ -555,44 +575,53 @@ open class GenericHPPManager<T: Decodable>: NSObject, UIWebViewDelegate, HPPView
         self.delegate?.HPPManagerCancelled!()
         self.genericDelegate?.HPPManagerCancelled()
     }
-
 }
-extension NSDictionary {
-	//: ### Base64 encoding a string
 
-	func DecodeAllValues() -> NSMutableDictionary
-	{
-		let dic: NSMutableDictionary! = NSMutableDictionary.init(capacity: self.count)
-		for value in self {
-			if (value.value as? String) != ""
-			{
-				dic[value.key] = (value.value as?String)?.base64Decoded()
+extension NSDictionary {
+    /// Decoded array values in Base64
+    func decodeAllValues() -> NSMutableDictionary {
+        let dict: NSMutableDictionary! = NSMutableDictionary(capacity: self.count)
+
+        for value in self {
+            if (value.value as? String) != "" {
+                dict[value.key] = (value.value as?String)?.base64Decoded()
+            } else {
+                dict[value.key] = value.value
+            }
+        }
 
-			}
-			else{
-				dic[value.key] = value.value
+        return dict
+    }
+}
 
-			}
-		}
+extension String {
 
-		return dic
-	}
+    /// Encoded string in Base64
+    func base64Encoded() -> String? {
+        guard let data = self.data(using: .utf8) else { return nil }
+        return data.base64EncodedString()
+    }
 
+    /// Encoded string in Base64
+    func base64Decoded() -> String? {
+        guard let data = Data(base64Encoded: self) else { return nil }
+        return String(data: data, encoding: .utf8)
+    }
 }
-extension String
-{
-	func base64Encoded() -> String? {
-		if let data = self.data(using: .utf8) {
-			return data.base64EncodedString()
-		}
-		return nil
-	}
-
-	//: ### Base64 decoding a string
-	func base64Decoded() -> String? {
-		if let data = Data(base64Encoded: self) {
-			return String(data: data, encoding: .utf8)
-		}
-		return nil
-	}
+
+private struct HPPHeader {
+
+    struct Field {
+        static let contentType = "Content-Type"
+        static let accept = "Accept"
+        static let userAgent = "User-Agent"
+        static let lclRequest = "X-LCL-Request"
```

Comment:
Again, LCL in the SDK, you did this as requested but once again I should have pushed back on this one. Let me hit the business up and see what value will suffice here.